packages/node_modules/@node-red/runtime/lib/index.js

```diff
@@ -396,6 +398,8 @@ module.exports = {
     */
     version: externalAPI.version,
 
+    module: npmModule,
```

Comment:
If this is part of the external runtime API, then it must go via a file in `externalAPI`.

I also have trouble with the name `module` - this is a very specific case of generating a tgz for a subflow. Module does not seem like the right name - same feedback as with the url.

I think this should be called `subflows` - and will become the home of a general subflow api in the future.

Reply:
I assumed that there would be other things in the future besides SUBFLOW that could be downloaded as NPM modules.  So  I used `module`.
But changed it to `subflows` in update.